### PR TITLE
Darker highlight color, dark search bar, colored buttons

### PR DESCRIPTION
**Pull request by KiitoX**
Fixes MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1188 and #39

### CODE REVIEW

**Comment by KiitoX** [comment]
Some sample images:
Search bar:
bar
Coloured about artist button:
dark
light
Darker highlight colour:
list
sidebar

File: src/gmusic-theme.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const CONSTANTS = require('../lib/_constants');
 const DEFAULTS = {
   BACK_PRIMARY: '#222326',
   BACK_SECONDARY: '#121314',
-  BACK_HIGHLIGHT: '#615F59',
+  BACK_HIGHLIGHT: '#1a1b1d',
```
Comment (MarshallOfSound) [inline src/gmusic-theme.js]: Let's leave this in GPMDP only for now.  Making this change and following semver (like I do) would require this to be a breaking major version jump.  And I hate those

File: lib/full.css [hunk 2]
```diff
@@ -14,6 +14,29 @@
   color: <<FORE_PRIMARY>>;
 }
 
+.sj-search-box-0,
```
Comment (MarshallOfSound) [inline lib/full.css]: Can we somehow use selectors that don't use `-X`

Those selectors tend to be the ones that break the most frequently

Comment (MarshallOfSound) [inline lib/full.css]: For instance if we used to DOM tag selector instead of the class

`sj-search-box`

File: dist/gmusic-theme.js [hunk 3]
```diff
@@ -229,9 +229,9 @@ window.GMusicTheme = GMusicTheme;
 
 
 },{"../lib/_constants":2,"color":7}],2:[function(require,module,exports){
-module.exports = {
-  CLASS_NAMESPACE: 'gmusic-theme',
-};
+module.exports = {
```
Comment (MarshallOfSound) [inline dist/gmusic-theme.js:232]: I love git...

Nothing changed, yet something changed 😆

Comment (KiitoX) [inline dist/gmusic-theme.js:232]: hmmm.... lemme build it again...

Comment (KiitoX) [inline dist/gmusic-theme.js:232]: yeah no, no idea I did nothing...

Comment (MarshallOfSound) [inline dist/gmusic-theme.js:232]: All good, probably line endings.  I'll rebuild on my PC before release anyway 😆

Comment (KiitoX) [inline dist/gmusic-theme.js:232]: What ending do you use? I'm using LF so it could be that...

**Comment by MarshallOfSound** [comment]
LGTM 👍

File: dist/gmusic-theme.js (hunk 3 above)
Comment (MarshallOfSound) [inline dist/gmusic-theme.js:232]: I always commit LF, but we've had this issue before with @chrismou 

So I just ignore it from now on 😆

**Pull request merged**